recipes/kmedoids/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "kmedoids" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
```

```suggestion
{% set version = "0.4.1" %}

package:
  name: kmedoids
```
This variable really adds a lot of indirection. Its mostly there to help pull the package more easily from pypi, but honestly, i like to just hardcode the package name most places since the "pattern" is really not that useful to read.